src/sorcha_cmdline/run.py

```diff
@@ -0,0 +1,197 @@
+#
+# The `sorcha run` subcommand implementation
+#
+import argparse
+
+def main():
+    parser = argparse.ArgumentParser(formatter_class=argparse.ArgumentDefaultsHelpFormatter)
```

Comment:
Is there any help text for this subcommand anywhere? I know it's fairly self-explanatory, but I wonder do we want a couple of words like "run main Sorcha simulation".